lib/integration/scenarios/Test/Integration/Scenario/API/Byron/Migrations.hs

```diff
@@ -34,6 +33,9 @@ import Cardano.Wallet.Api.Types
 import Cardano.Wallet.Api.Types.Amount
     ( ApiAmount (ApiAmount)
     )
+import Cardano.Wallet.Api.Types.Era
+    ( ApiEra (..)
```

Comment:
yeah, why not to import `toAnyCardanoEra` here ???

Reply:
> yeah, why not to import `toAnyCardanoEra` here ???

Because now we have `ApiEra.toAnyCardanoEra`, which mentions **_both_** the source type `ApiEra` **_and_** the target type `AnyCardanoEra`.

This is analogous to `Map.toList` being a function exported from `Data.Map` into the `Map` namespace. By writing `Map.toList`, it makes it clear to readers that this a conversion from a `Map` to a list.